euphony/src/main/java/co/euphony/util/Constants.kt

```diff
@@ -0,0 +1,10 @@
+package co.euphony.util
+
+object Constants {
+
+    // TxRxChecker Test Tags
+    const val TAG_INPUT = "TxRxCheckerInput"
+    const val TAG_BUTTON = "TxRxCheckerButton"
+    const val TAG_RESULT_SUCCESS = "TxRxCheckerResultSuccess"
+    const val TAG_RESULT_FAIL = "TxRxCheckerResultFail"
+}
```

Comment:
We already have Constants class in `co.euphony.common` package.
So, those string constants should be merged with `co.euphony.common.Constants`